### PR TITLE
HTML a few small changes, read below or commits

### PR DESCRIPTION
**Pull request by car2t**
Hi all 

I did this in the HTML document.

1 - Add alt attribute with an empty string value to all the images

2 - Switch the p.footer-text for an address element

3 - Delete final periods in the footer. Visually annoy, since it looked like headings with a final dot.

4 - I have added a `div `wrapping the whole page: `<div id="web-app-wrapper">`. 
    I want to try putting in the general layout and making the body a small grid container using `grid-template-columns: 1rem 1fr 1rem;`. I think it will look better.

I hope you like it.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -1,15 +1,11 @@
...
     <meta http-equiv="Content-type" content="text/html; charset=UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
     <!-- displays site properly based on user's device -->
-    <link
-      rel="icon"
-      type="image/png"
-      sizes="32x32"
-      href="./images/favicon-32x32.png"
-    />
+    <link rel="icon" type="image/png" sizes="32x32" href="./images/favicon-32x32.png" />
```
Comment (WitchDevelops) [inline index.html:8]: only formatting, not really necessary

File: index.html [hunk 2]
```diff
@@ -18,38 +14,37 @@
   </head>
 
   <body>
-    <main>
-      <div id="main-card">
-        <h1>Advice #</h1>
-        <p id="advice">
-          "<span id="advice-text"><!-- Advice text goes here --></span>"
+    <div id="web-app-wrapper">
```
Comment (WitchDevelops) [inline index.html:17]: why

**Comment by WitchDevelops** [comment]
> 1 - Add alt attribute with an empty string value to all the images
I don't see this change. There is `alt` on the dice (not an empty string) and no `alt` on the divider.

> 2 - Switch the p.footer-text for an address element
It's a good idea, but I don't see this in the final code.

> 3 - Delete final periods in the footer. Visually annoy, since it looked like headings with a final dot.
ok, I'm fine with that

> 4 - I have added a div wrapping the whole page: <div id="web-app-wrapper">.
I want to try putting in the general layout and making the body a small grid container using grid-template-columns: 1rem 1fr 1rem;. I think it will look better.
I don't understand the reason why do you want to do this. Can you explain why do you think it's a good idea? To put the whole `body` in a `div` element? If you want to make it into a grid container, why not declare it on the `body`?

**Review by WitchDevelops — CHANGES_REQUESTED** [review]
Please provide a reason for those changes :)

**Comment by car2t** [comment]
Hi Domi

> > 1 - Add alt attribute with an empty string value to all the images
> 
**> I don't see this change. There is `alt` on the dice (not an empty string) and no `alt` on the divider.** 

_It looks that I have missed a few. Oops! Yeah! You are right, maybe we can add alt="" to dividers and more descriptive one to the image. 
Also I have forgot about adding `width` and `height` attributes too; better give it in advance to the browser_

> > > 2 - Switch the p.footer-text for an address element
> 
> **It's a good idea, but I don't see this in the final code.**

_Ok, never mind, not important._

> > 3 - Delete final periods in the footer. Visually annoy, since it looked like headings with a final dot.
> 
> **ok, I'm fine with that** _Cool!_ ;D
> 
> > 4 - I have added a div wrapping the whole page: .
> > I want to try putting in the general layout and making the body a small grid container using grid-template-columns: 1rem 1fr > 1rem;. I think it will look better.
> 
> **I don't understand the reason why do you want to do this. Can you explain why do you think it's a good idea? To put the whole `body` in a `div` element? If you want to make it into a grid container, why not declare it on the `body`?**

_I can test on my own, not important, we can leave it as it is. No problemo._

Thank you Domi for taking your time to review this. :D

**Comment by car2t** [comment]
I will wait for others speak and see what new things we can do. 
Also I probably will delete commits or directly the branch, not necessary for now, right?

By for now

**Comment by caesiumtea** [comment]
@car2t 
It looks like your final commit in the `html` branch (commit 98245a2cb8e101fef36203b5b41d81c0fdb7f993) actually undoes the changes in your previous couple of commits! I suggst we reset the `html` branch to commit 25a7beca0f089903b5342616e8682f421a1dd0cd and then open a PR on that.

> 1 - Add alt attribute with an empty string value to all the images

Good catch, yes, those dividers should definitely have `alt=""` on them! But don't change the alt text on the dice button--that one is NOT purely decorative. The alt text should indicate that it serves a "randomize" function, because that's the meaning that the dice symbol expresses.

> 2 - Switch the p.footer-text for an address element

Sounds good to me
 
> 3 - Delete final periods in the footer. Visually annoy, since it looked like headings with a final dot.

Sure, I agree it looked a little odd.

> 4 - I have added a `div `wrapping the whole page: `<div id="web-app-wrapper">`. I want to try putting in the general layout and making the body a small grid container using `grid-template-columns: 1rem 1fr 1rem;`. I think it will look better.

Go ahead and try it out, because I can't figure out what difference it will make just by imagining it haha. But maybe in a separate branch from these alt text and footer changes.

@WitchDevelops 
> I don't understand the reason why do you want to do this. Can you explain why do you think it's a good idea? To put the whole body in a div element? If you want to make it into a grid container, why not declare it on the body?

Is it just because it's good to keep semantic elements separate from style-related elements? idk

**Pull request closed without merging**